README.markdown

```diff
@@ -1065,3 +1098,27 @@ The second exception has to do with file and path names. In order to accommodate
 
 Finally, I have done my best to accommodate non-ASCII characters in logs. I cannot promise that everything works, but I'd like to know if you see issues with this.
  
+### Equation preview shows "ERROR: Failed to run 'pdflatex' to create pdf to preview."

```

Comment:
I am preparing a PR to improve and open the error report so this section might not be necessary.